pyhuman/app/workflows/browse_youtube.py

```diff
@@ -0,0 +1,80 @@
+from time import sleep
+import os
+import random
+
+# from soupsieve import select
+
+from ..utility.base_workflow import BaseWorkflow
+from ..utility.webdriver_helper import WebDriverHelper
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.common.exceptions import ElementNotInteractableException
+
+WORKFLOW_NAME = 'YoutubeBrowser'
+WORKFLOW_DESCRIPTION = 'Browse Youtube'
+
+DEFAULT_INPUT_WAIT_TIME = 2
+MAX_WATCH_TIME = 150 # Maximum amount of time to watch a video, in seconds
+MAX_WAIT_TIME = 5 # Maximum amount of time to wait after searching, in seconds
+MAX_SUGGESTED_VIDEOS = 10
+
+SEARCH_LIST = 'browse_youtube.txt'
+
+def load():
+    driver = WebDriverHelper()
+    return GoogleSearch(driver=driver)
+
+
+class GoogleSearch(BaseWorkflow):
+
+    def __init__(self, driver, input_wait_time=DEFAULT_INPUT_WAIT_TIME):
+        super().__init__(name=WORKFLOW_NAME, description=WORKFLOW_DESCRIPTION, driver=driver)
+
+        self.input_wait_time = input_wait_time
+        self.search_list = self._load_search_list()
+
+    def action(self, extra=None):
+        self._search_web()
+
+    """ PRIVATE """
+
+    def _search_web(self):
+        random_search = self._get_random_search()
+
+        # Navigate to youtube
+        self.driver.driver.get('https://www.youtube.com')
+        sleep(random.randrange(2, MAX_WAIT_TIME))
```

Comment:
May also be a good idea to save the minimum wait time to a constant called MIN_WAIT_TIME.